projects/view/client/rtProject/rtProjectInfoSection.css

```diff
@@ -0,0 +1,215 @@
+.rtProjectInfoSection .section { 
+	max-width: 1024px;
+	clear: both;
+	padding: 0px;
+	margin: 0px;
+	margin-left: auto;
+	margin-right: auto;
+}
+.rtProjectInfoSection .content{ 
+	min-height: 300px;
+	display: table;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.content{ 
+		min-height: 0px;
+	}	
+}
+.rtProjectInfoSection .content .col{ 
+	display: table-cell;
+	vertical-align: middle;
+	float: none;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.content .col{ 
+		display: none;
+		min-height: 200px;
```

Comment:
It looks like Cloud9 allows workspaces to have custom whitespace settings. In the bottom right corner of the code editor, there is a configuration menu:

![screenshot](https://cloud.githubusercontent.com/assets/17307/13001389/061cfe08-d16e-11e5-9a4a-eea6aa27e22b.png)